bedrock/base/templates/includes/banners/vpn-coupon-variations/vpn-coupon-promo-1.html

```diff
@@ -0,0 +1,102 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+<aside class="c-banner hide-from-legacy-ie" id="vpn-coupon-promo-banner">
+  <div class="c-banner-inner">
+    <div class="c-coupon-wrapper c-coupon">
+
+      {% if LANG == 'de' %}
+      <div class="c-coupon-title-wrapper">
+
+        <h2 class="c-coupon-title">
+          Sicher durchs Netz
```

Comment:
You could set all of these strings as variables in one place at the top like you did with the `link_text` and avoid repeating all the markup. Something like:

`{% if LANG == 'de' %}`
`{% set banner_title = 'Sicher durchs Netz' %}`

With other conditions for English and French (probably make English the `else` state so it's the default). Then you can display that variable in the template with `{{ banner_title }}`